pkg/gatewayserver/io/mqtt/mqtt.go

```diff
@@ -44,6 +46,9 @@ type srv struct {
 	lis    mqttnet.Listener
 }
 
+// ErrMQTTFrontendRecovered is returned when a panic is caught from the MQTT frontend.
+var ErrMQTTFrontendRecovered = errors.DefineInternal("mqtt_frontend_recovered", "Internal Server Error")
```

Comment:
- we don't expose errors
- is the error that it's recovered, or that there's an internal server error?
- no title case in error messages

Reply:
basically adapted from our [echo middleware](https://github.com/TheThingsNetwork/lorawan-stack/blob/c83dcc8f6aa77a886768723a4963169f509ce46e/pkg/web/middleware/recover.go#L30).

Reply:
> we don't expose errors

I'm not actually returning this anywhere right? It's only printing to `stderr` and logs it.

Reply:
OK for consistency, although I don't really like it there

It's package exposed, i.e. `Err...` instead of `err...`